.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: "pre-release"
+
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  pre-release:
+    name: "Pre Release"
+    runs-on: "ubuntu-latest"
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: clone
+      run: git submodule update --init --recursive
+    - name: install toolchain
+      run: curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q1-update/+download/gcc-arm-none-eabi-5_3-2016q1-20160330-linux.tar.bz2" | tar xfj -
```

Comment:
Isn't this really outdated? The docs say we're targeting 7.3.1, and that's what we get using the PPA (also what's in the developer docs). Should we switch to a PPA install here? Might also need to update the CI actions